### ISSUE TITLE
object no longer allowed on Schema attributes

### ISSUE THREAD

**Issue by bobvandevijver** [zircote/swagger-php#1235]
https://github.com/zircote/swagger-php/pull/1217 removed the possibility to pass an object to the `ref` parameter of the `Items` class, which breaks the following usage in combination with NelmioApiDocBundle:

```php
use Nelmio\ApiDocBundle\Annotation\Model;
use OpenApi\Attributes as OA;

class EquipmentInformation
{
  private int $id;

  private string $name;

  #[OA\Property(nullable: true)]
  private ?string $code;

  #[OA\Property(type: 'array', items: new OA\Items(ref: new Model(type: SubEquipmentInformation::class)))]
  private array $subEquipment;
}
```

It looks like the same was done for the others classes that extend the Schema annotation, which basically renders https://github.com/nelmio/NelmioApiDocBundle/blob/master/OpenApiPhp/ModelRegister.php#L44 useless and thus breaks the current usages as the one above.

Was this change intentional, and if so, is there another way of achieving the same? Maybe by using the `attachables`, which means the Nelmio bundle needs to be update so their classes implement the `Attachable` class? 

Changing this particular example to one of the following all break:

```
#[OA\Property(type: 'array', items: new OA\Items(attachables: [new Model(type: SubEquipmentInformation::class)]))]
#[OA\Property(type: 'array', attachables: [new Model(type: SubEquipmentInformation::class)]))]
```

This might be related to https://github.com/nelmio/NelmioApiDocBundle/issues/1990 as well.

**Comment by DerManoMann** [comment]
This was fixed in 4.4.1. Updating to latest swagger-php should make things work again.

**Comment by bobvandevijver** [comment]
Alright, that is definitely my bad... Why didn't I check the releases? 😔
